dm/config/task.go

```diff
@@ -310,6 +312,7 @@ func NewTaskConfig() *TaskConfig {
 		Mydumpers:               make(map[string]*MydumperConfig),
 		Loaders:                 make(map[string]*LoaderConfig),
 		Syncers:                 make(map[string]*SyncerConfig),
+		RemoveFinishedDump:      false,
```

Comment:
How about removing dumped data by default?

Reply:
I'll try setting true by default and check if there's any broken test.

Reply:
default `true` can't pass some tests, checking again

https://internal.pingcap.net/idc-jenkins/blue/organizations/jenkins/dm_ghpr_test/detail/dm_ghpr_test/5490/pipeline/81/